dev/ci/ci-common.sh

```diff
@@ -35,10 +36,10 @@ ls "$COQBIN"
 CI_BUILD_DIR="$PWD/_build_ci"
 
 # shellcheck source=ci-basic-overlay.sh
-source "${ci_dir}/ci-basic-overlay.sh"
+. "${ci_dir}/ci-basic-overlay.sh"
```

Comment:
Replacing source with . : technically fine, but reusable shell scripts should be optimized for readability. I use . interactively and source in multi-use scripts, and I think many follow this approach.

Applies to all other identical changes

Reply:
Actually, I tend to replace `source` by `.` because `.` is posix and `source` is not. Here we're using bash, so `source` is OK but still... I usually prefer posix things when possible. I think `.` is well known and so it seems fine to me to use it. Anyway, if you still want me to change it, will do.

Reply:
Interesting. I used source since ever also on non Linux / GNU based Unixes (IRIX, AIX, HP-UX) but I can confirm that it is not Posix, so I agree that . is preferable.